### DISCUSSION TITLE
Deckgl layers dont work with MobX

### DISCUSSION THREAD

**Discussion by Pozner37** [visgl/deck.gl#9129] [Q&A]
Hello,
Im trying to give deckgl layers data prop which is a mobx observable map or array and it doesnt work , the layer isnt drawing the data. But when i give the layer my observable array .toJson() it works.
So im guessing there is an issue with deckgl that it doesnt know how to handle mobx observables since they are not plain js they use proxies and observables.
Any help?

**Comment by Pessimistress** [comment]
Can you provide a Codepen/JS Fiddle that reproduces your issue?
